Content.Shared/Atmos/Atmospherics.cs

```diff
@@ -271,6 +271,21 @@ static Atmospherics()
         public const float MaxTransferRate = 200;
 
         #endregion
+
+        /// <summary>
+        ///     Color to represent the gas in UIs and such.
+        ///     Currently only used in the debug overlay.
+        ///
+        ///     Ordering needs to be the same as <see cref="Gas"/>.
+        /// </summary>
+        public static readonly Color[] GasDisplayColors = {
```

Comment:
this needs to have a size of `Atmospherics.TotalNumberOfGases` (and probably some fallback color) so no out of bounds accesses if someone adds a new gas and forgets to update this